src/lib/services/feature-toggle-service.ts

```diff
@@ -1279,7 +1279,7 @@ class FeatureToggleService {
             )
         ) {
             throw new Error(
-                `Change requests are enabled for ${project} in ${environment} environment`,
+                `The change requests are enabled in project ${project} for environment ${environment}`,
```

Comment:
Hi! Nerd fairy swiping by 🧚🏼  I may be missing some context, but I don't _think_ that this sentence should start with "the". Also, a minor suggestion to wrap project and environment names in quotes for clarity, but feel free to dismiss that 🤓 

```suggestion
                `Change requests are enabled in project "${project}" for environment "${environment}"`,
```